src/Components/Header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import logo from '../../Assets/Images/logo.png';
+import './Header.scss';
```

Comment:
do not import sccss fo every separate component.
IMport index.scss inside app.jsx file
and inside index.scss file import other scss files